Fitbit.Portable.Tests/SleepTests.cs

```diff
@@ -47,7 +48,7 @@ public class SleepTests
 
             var responseMessage = new Func<HttpResponseMessage>(() =>
             {
-                return new HttpResponseMessage(HttpStatusCode.OK) { Content = new StringContent(content) };
```

Comment:
No need to remove the spaces; inconsistent with below

Reply:
Noted